### ISSUE TITLE
memleak on RAI_ScriptRunCtxCreate RAI_ScriptRunCtx* sctx = RedisModule_Calloc(1, sizeof(*sctx));

### ISSUE THREAD

**Issue by filipecosta90** [RedisAI/RedisAI#292]
leak on 
``` c
RAI_ScriptRunCtx* sctx = RedisModule_Calloc(1, sizeof(*sctx)); 
```

```
==28766== 6,549 (80 direct, 6,469 indirect) bytes in 2 blocks are definitely lost in loss record 132,389 of 132,415
==28766==    at 0x4C31B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==28766==    by 0x144DB6: zcalloc (zmalloc.c:131)
==28766==    by 0x1C6FAC: RM_Calloc (module.c:335)
==28766==    by 0x67EC594: RAI_ScriptRunCtxCreate (script.c:110)
==28766==    by 0x67DAE87: RedisAI_ScriptRun_RedisCommand (redisai.c:1170)
==28766==    by 0x1C76BE: RedisModuleCommandDispatcher (module.c:575)
==28766==    by 0x13D315: call (server.c:2468)
==28766==    by 0x13DFCA: processCommand (server.c:2763)
==28766==    by 0x14F086: processInputBuffer (networking.c:1470)
==28766==    by 0x14F1DB: processInputBufferAndReplicate (networking.c:1505)
==28766==    by 0x14F551: readQueryFromClient (networking.c:1587)
==28766==    by 0x1342E2: aeProcessEvents (ae.c:443)
```

**Comment by lantiga** [comment]
Fixed in #289 (see caa8cab)

**Comment by filipecosta90** [comment]
leak fixed :+1:
